src/global.d.ts

```diff
@@ -41,6 +41,6 @@ interface IGraphData {
 
 type NodeClickFunc = (node: INode) => void;
 
-type GraphType = 'echarts' | 'antv';
+type GraphType = 'echarts';
```

Comment:
Can we safely remove this `GraphType`, since there is only one graph type? 
BTW, do we have to remove the `graphType` options in views? Maybe it could be useful again in the future.

Reply:
I think it will be safe if we remove all `graphType`, `GraphType` in the codebase. But there are also some trivial code refactoring work being involved.

I can handle this, and I will push a commit to this PR branch after removing them all, safely.